egs/zeroth_korean/s5/local/online/run_nnet2_ms.sh

```diff
@@ -0,0 +1,267 @@
+		--feat-type raw \
+		--online-ivector-dir exp/nnet2_online/ivectors_train_2x_hires \
+		--cmvn-opts "--norm-means=false --norm-vars=false" \
+		--num-threads "$num_threads" \
+		--minibatch-size "$minibatch_size" \
+		--parallel-opts "$parallel_opts" \
+		--io-opts "--max-jobs-run 12" \
+		--initial-effective-lrate 0.0015 --final-effective-lrate 0.00015 \
+		--cmd "$decode_cmd" \
+		--pnorm-input-dim 3500 \
+		--pnorm-output-dim 350 \
+		--mix-up 12000 \
+		data/train_2x_hires data/lang exp/tri5b $dir  || exit 1;
+fi
+
+#if [ $stage -le 8 ]; then
+#  echo "#### $0: stage 8 #####"
+#  # dump iVectors for the testing data.
+#  for test in test_clean; do
+#    steps/online/nnet2/extract_ivectors_online.sh --cmd "$train_cmd" --nj 10 \
+#      data/${test}_hires exp/nnet2_online/extractor exp/nnet2_online/ivectors_$test || exit 1;
+#  done
+#fi
+
+#if [ $stage -le 9 ]; then
+#  # this does offline decoding that should give about the same results as the
+#  # real online decoding (the one with --per-utt true)
+#  for test in testData01; do
+#    steps/nnet2/decode.sh --nj 10 --cmd "$decode_cmd" --config conf/decode.config \
+#      --online-ivector-dir exp/nnet2_online/ivectors_${test} \
+#      exp/tri5b/graph_tgsmall data/${test}_hires $dir/decode_${test}_tgsmall || exit 1;
+#    steps/lmrescore.sh --cmd "$decode_cmd" data/lang_test_{tgsmall,tgmed} \
+#      data/${test}_hires $dir/decode_${test}_{tgsmall,tgmed}  || exit 1;
+#    steps/lmrescore_const_arpa.sh \
+#      --cmd "$decode_cmd" data/lang_test_{tgsmall,tglarge} \
+#      data/$test $dir/decode_${test}_{tgsmall,tglarge} || exit 1;
+#    steps/lmrescore_const_arpa.sh \
+#      --cmd "$decode_cmd" data/lang_test_{tgsmall,fglarge} \
+#      data/$test $dir/decode_${test}_{tgsmall,fglarge} || exit 1;
+#  done
+#  for test in testData02; do
+#    steps/nnet2/decode.sh --nj 4 --cmd "$decode_cmd" --config conf/decode.config \
+#      --online-ivector-dir exp/nnet2_online/ivectors_${test} \
+#      exp/tri5b/graph_tgsmall data/${test}_hires $dir/decode_${test}_tgsmall || exit 1;
+#    steps/lmrescore.sh --cmd "$decode_cmd" data/lang_test_{tgsmall,tgmed} \
+#      data/${test}_hires $dir/decode_${test}_{tgsmall,tgmed}  || exit 1;
+#    steps/lmrescore_const_arpa.sh \
+#      --cmd "$decode_cmd" data/lang_test_{tgsmall,tglarge} \
+#      data/$test $dir/decode_${test}_{tgsmall,tglarge} || exit 1;
+#    steps/lmrescore_const_arpa.sh \
+#      --cmd "$decode_cmd" data/lang_test_{tgsmall,fglarge} \
+#      data/$test $dir/decode_${test}_{tgsmall,fglarge} || exit 1;
+#  done
+#fi
+
+
+if [ $stage -le 10 ]; then
+  echo "#### $0: stage 10 #####"
+  # If this setup used PLP features, we'd have to give the option --feature-type plp
+  # to the script below.
+  steps/online/nnet2/prepare_online_decoding.sh --mfcc-config conf/mfcc_hires.conf \
+    data/lang exp/nnet2_online/extractor "$dir" ${dir}_online || exit 1;
+fi
+
+#if [ $stage -le 11 ]; then
```

Comment:
There is a lot of commented stuff here.  It feels like you have just added whatever was lying around in a directory to a PR, without really checking it.